Sources/BraintreeVenmo/BTVenmoAnalytics.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class BTVenmoAnalytics {
+    
+    // MARK: - Conversion Events
+    
+    static let tokenizeStarted = "venmo:tokenize:started"
+    static let tokenizeFailed = "venmo:tokenize:failed"
+    static let tokenizeSucceeded = "venmo:tokenize:succeeded"
+    
+    // MARK: - Additional Detail Events
+    
+    static let tokenizeNetworkConnectionLost = "venmo:tokenize:network-connection:failed"
+    static let appSwitchSucceeded = "venmo:tokenize:app-switch:succeeded"
+    static let appSwitchFailed = "venmo:tokenize:app-switch:failed"
+    static let appSwitchCanceled = "venmo:tokenize:app-switch:canceled"
```

Comment:
The canceled case is a `conversion` event, so it should move up.

Reply:
Another thought I was having, maybe the generic cancels would be more clear as `venmo:tokenize:canceled`, and have that syntax apply to all modules. So `<feature>:<method_name>:canceled`.

We can tackle this in a follow up PR though if we like it.

Reply:
Great catch with that. Will change it. If we don't need specifics of how/where cancel happened with these anlytic messages, I am fine to change it later.